spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,26 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before do
+      get user_posts_path(user_id: 1)
+    end
+
+    it 'returns correct status' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('posts/index')
+    end
+
+    it 'returns the correct placeholder text' do
+      expect(response.body).to include('Here is a list of posts')
+    end
+  end
+
+  describe 'GET /show' do
+    before do
+      get user_post_path(user_id: 1, id: 1)
+    end
```

Comment:
- [ ] [Optional] [Highly Recommended] Please make sure that for `posts show`  you add tests that check:

- If the response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️

Reply:
Fixed.